stage3.Dockerfile

```diff
@@ -7,24 +7,27 @@
 # to fetch the signing key.
 
 ARG BOOTSTRAP
-FROM ${BOOTSTRAP:-alpine:3.6} as builder
+FROM ${BOOTSTRAP:-alpine:3.7} as builder
 
 WORKDIR /gentoo
 
 ARG ARCH=amd64
 ARG MICROARCH=amd64
 ARG SUFFIX
-ARG DIST="http://distfiles.gentoo.org/releases/${ARCH}/autobuilds/"
+ARG DIST="https://ftp-osl.osuosl.org/pub/gentoo/releases/${ARCH}/autobuilds"
 ARG SIGNING_KEY="0xBB572E0E2D182910"
 
 RUN echo "Building Gentoo Container image for ${ARCH} ${SUFFIX} fetching from ${DIST}" \
  && apk --no-cache add gnupg tar wget \
- && STAGE3PATH="$(wget -q -O- "${DIST}/latest-stage3-${MICROARCH}${SUFFIX}.txt" | tail -n 1 | cut -f 1 -d ' ')" \
+ && STAGE3PATH="$(wget -O- "${DIST}/latest-stage3-${MICROARCH}${SUFFIX}.txt" | tail -n 1 | cut -f 1 -d ' ')" \
+ && echo "STAGE3PATH:" $STAGE3PATH \
```

Comment:
is this echo necessary or debug?

Reply:
Debug, but handy.  I thought about taking it out for the commit, but it is a very handy piece of logging output for the build, otherwise there is no record in the Docker or Travis output of where stage3 image was fetched-from.

Can remove if preferred.